Cargo.toml

```diff
@@ -8,6 +8,17 @@ edition = "2021"
 [dependencies]
 webthing = "0.15.0"
```

Comment:
shouldn't it be removed at the same time?

Reply:
Yes, I totally did not update Cargo.toml. Probably there are other things that can be safely removed as well.